src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -776,4 +783,25 @@ public static int getPoLineLimit(JsonObject config) {
   public static CompositePurchaseOrder convertToCompositePurchaseOrder(JsonObject poJson) {
     return poJson.mapTo(CompositePurchaseOrder.class);
   }
+
+  public static boolean inventoryUpdateNotRequired(CompositePoLine compPOL) {
+    return (compPOL.getPhysical() == null && compPOL.getEresource() == null) ||
+      (compPOL.getPhysical() != null && compPOL.getPhysical().getCreateInventory() == Physical.CreateInventory.NONE && compPOL.getEresource() == null) ||
+      (compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == Eresource.CreateInventory.NONE && compPOL.getPhysical() == null) ||
+      (compPOL.getPhysical() != null && compPOL.getEresource() != null && compPOL.getEresource().getCreateInventory() == Eresource.CreateInventory.NONE && compPOL.getPhysical().getCreateInventory() == Physical.CreateInventory.NONE) ||
+      // also ignore inventory interaction in case of "Other" order format
```

Comment:
I think there's a misunderstanding here... We don't want to ignore inventory interaction for "Other" orders.  

In the case of "Other", use createInventory and quantity from poLine.physical.  The JIRA describes that the UI should set the default value of physical.createInventory to 'None', but there's nothing preventing a user from changing the drop-down value.

Reply:
Please make sure test cases are updated to cover this too

Reply:
Updated logic for "Other" format.